tools/android/build_aar.py

```diff
@@ -77,12 +74,12 @@ def do_build_by_mode(
 
             jnilib_names = ["libortextensions.so", "libonnxruntime_extensions4j_jni.so"]
             for jnilib_name in jnilib_names:
-                shutil.copyfile(build_dir / config / "java" / "android" / abi / jnilib_name, jnilibs_dir / jnilib_name)
+                shutil.copyfile(build_dir / config / "lib" / jnilib_name, jnilibs_dir / jnilib_name)
```

Comment:
No clear that we want this change to the path. Something changed in the `main` CI that required it (possibly the gradle version of the image being used to build but we haven't had time to figure out what it was), but the 0.19 branch was still passing without it. 

could try with it first to keep the cherry pick clean and undo if needed.